src/app/app-routing.module.ts

```diff
@@ -1,5 +1,6 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import { BrowserUtils } from '@azure/msal-browser';
```

Comment:
Cab be removed if not used.

Reply:
I have removed unused imports and PR should target to main. Thanks.